sharktank/tests/layers/sharded_paged_kv_cache_test.py

```diff
@@ -151,7 +151,7 @@ def testWriteTimestep(self):
         cache_partitions = [
             torch.rand(
                 self.batch_size,
-                self.block_seq_len * self.block_seq_stride,
+                1,
```

Comment:
Any reason we decreased this to one?

Reply:
This was @sogartar changes to fix the shard tests issues after scatter fusion.

Reply:
It was wrong. The expectation is that this dimension should be always size 1. That is at least the comment at the [function defintion](https://github.com/nod-ai/SHARK-Platform/blob/8e074b45362737787e5ad1d467f0286976490e4f/sharktank/sharktank/layers/kv_cache.py#L348).